client/README.md

```diff
@@ -36,10 +38,26 @@ npm run build
 npm run start
 ```
 
+To run Avocart version of the client.
```

Comment:
Do you want to add anything about what the two versions are?

Reply:
@glasnt Added more color! Let me know if I forgot an item you're also doing to deploy.